pulseapi/entries/views.py

```diff
@@ -197,7 +198,67 @@ def get_queryset(self):
         bookmarks = UserBookmarks.objects.filter(user=user)
         return Entry.objects.filter(bookmarked_by__in=bookmarks).order_by('-bookmarked_by__timestamp')
 
-    serializer_class = EntrySerializer
+    # When people POST to this route, we want to do some
```

Comment:
This is going to need an update to the README.md too, to explain that this route exists, how to call it, and what can be expected back =)